test/CreatorToken.t.sol

```diff
@@ -264,11 +287,104 @@ abstract contract Buying is CreatorTokenTest {
 }
 
 abstract contract Selling is CreatorTokenTest {
-  function test_sellAToken(address _seller) public {
+  function test_SellAToken(address _seller) public {
     buyAToken(_seller);
     sellAToken(_seller, creatorToken.lastId());
   }
 
+  function test_BulkSell(address _seller, uint256 _numTokensToBuyAndSell) public {
+    _assumeSafeBuyer(_seller);
+    _numTokensToBuyAndSell = bound(_numTokensToBuyAndSell, 1, 100);
+    uint256[] memory _tokenIds = new uint256[](_numTokensToBuyAndSell);
+
+    uint256 _originalCreatorTokenBalanceOfSeller = creatorToken.balanceOf(_seller);
+    uint256 _originalPayTokenBalanceOfCreatorTokenContract =
+      payToken.balanceOf(address(creatorToken));
+    uint256 _originalCreatorTokenSupply = creatorToken.totalSupply();
+    uint256 _expectedNetProceeds;
+    uint256 _expectedPayTokenToBeEarnedByCreator;
+    uint256 _expectedPayTokenToBeEarnedByAdmin;
+
+    // buy n tokens
+    for (uint256 _i = 0; _i < _numTokensToBuyAndSell; _i++) {
+      buyAToken(_seller);
+      (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextSellPrice();
+      _expectedNetProceeds += _tokenPrice - _creatorFee - _adminFee;
```

Comment:
Very small nit, but when combining the `+=` operator with other operations on the right I prefer to use parens to make the order of ops explicit:

```solidity
_expectedNetProceeds += (_tokenPrice - _creatorFee - _adminFee);
```

Reply:
Updated in the same commit as above: [845a7c7](https://github.com/showtime-xyz/creator-tokens/pull/32/commits/845a7c78213feca63608e7a080180fa9b2a9abec)